pkgs/misc/lssecret/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv
+, pkg-config
+, libsecret
+, fetchFromGitLab
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  name = "lssecret";
+  version = "722013dc";
+
+  src = fetchFromGitLab {
+    owner = "GrantMoyer";
+    repo = name;
+    rev = version;
+    hash = "sha256-KLtxRXySoL9hT/c+O2arVR68ZYz+6Ce72YYCz60yot4=";
+  };
+
+  installPhase = "install -Dsm555 lssecret $out/bin/lssecret";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ libsecret ];
```

```suggestion
  nativeBuildInputs = [ pkg-config ];
  buildInputs = [ libsecret ];
  
  installPhase = "install -Dsm555 lssecret $out/bin/lssecret";
```
nit: ordering, inputs before phases.